buildkite/terraform/bazel/main.tf

```diff
@@ -1936,6 +1936,23 @@ resource "buildkite_pipeline" "bazel-bazel" {
   }
 }
 
+resource "buildkite_pipeline" "bazel-skymeld" {
+  name = "Bazel-Skymeld :bazel:"
+  repository = "https://github.com/bazelbuild/bazel.git"
+  steps = templatefile("pipeline.yml.tpl", { envs = {}, steps = { commands = ["curl -sS \"https://raw.githubusercontent.com/bazelbuild/continuous-integration/master/buildkite/bazelci.py?$(date +%s)\" -o bazelci.py", "python3.6 bazelci.py project_pipeline --file_config=.bazelci/postsubmit-skymeld.yml --monitor_flaky_tests=true | tee /dev/tty | buildkite-agent pipeline upload"] } })
```

Comment:
You can keep `--file_config=.bazelci/postsubmit-skymeld.yml`, but then you'll have to submit the yml file into bazelbuild/bazel. 

A simpler solution would be to use `--http_config=https://raw.githubusercontent.com/bazelbuild/continuous-integration/master/pipelines/postsubmit-skymeld.yml?$(date +%s)` here

Reply:
Makes sense, thanks for the suggestion. Done.